api_docs/changelog.md

```diff
@@ -22,15 +22,19 @@ format used by the Zulip server that they are interacting with.
 
 **Feature level 263**:
 
-* `POST /users/me/presence`: When `slim_presence` is used, a new
-  `presence_last_update_id` parameter can be given, instructing
+* `POST /users/me/presence`: A new `last_update_id`
```

Comment:
This was a small bug in this doc btw, the param is called `last_update_id` for this endpoint. (the prefix would be redundant here, since obviously it's in the `presence` domain so to speak; there isn't other non-presence data included here that would require such disambiguation via prefixes)